jupyter-book documentation

unmovingcastle opened the pull request.

Commit:
jupyter-book documentation initial commit

Commit:
finished initializing the jbook

Commit:
directory name change dummy -> jbook

Commit:
added github button

Commit:
small configuration change

branch: jbook -> gh-pages

ezio-melotti commented on GENETIS_doc/jbook/index.md:
```diff
@@ -1,3 +1,8 @@
+
+This documentation follows the
+[Diátaxis documentation guideline](https://diataxis.fr/)
+and is based on the dissertation by Dr. Julie Rolla, available
+[here](https://etd.ohiolink.edu/apexprod/rws_olink/r/1501/10?clear=10&p10_accession_num=osu163668789451345>).
```
```suggestion
and is based on the [dissertation by Dr. Julie Rolla](
https://etd.ohiolink.edu/apexprod/rws_olink/r/1501/10?clear=10&p10_accession_num=osu163668789451345).
```
The text of a link should be descriptive of what it's being linked, so "here" should be avoid whenever possible.

ezio-melotti commented on GENETIS_doc/.gitignore:
```diff
@@ -1,4 +1,5 @@
 *DS_Store
-docs/.vscode/
+*.vscode/
+*_build/
```
The `build` dir is still included, probably because of this:
```suggestion
*build/
```

Commit:
ignore sphinx build

Commit:
finished jupyter-book skeleton

Commit:
ready to make the transition from rst to MyST

Commit:
started modifying appendix a1

Commit:
finished rst2md for a1

Commit:
preparing to convert A3_1_1

Commit:
config quick fix

Commit:
finished part A conversion

Commit:
finished migrating from rst to mst

unmovingcastle committed:
Julie dissertation url change

Commit:
fix julie url

Commit:
added task list

Commit:
finished Monday work

Commit:
added bibliography and citation for ch3 and ch3_3

Commit:
Early investigations

started working on moving Early GENETIS Investigation (diss. ch. 3.2) to markdown

Commit:
early investigation complete

Commit:
finished discussion: symmetric bicone (3.4.1)
